samples/sampler/stories/helper/icons.js

```diff
@@ -30,13 +30,16 @@ export const listIcons = [
 ].sort();
 
 export const mediaIcons = [
+	'background',
```

Comment:
It seems that `background`, `eraser` and `pen` need to be added in drawingIcons.

Reply:
I updated it.